src/main/java/emu/grasscutter/server/dispatch/DispatchServer.java

```diff
@@ -177,23 +177,23 @@ public void start() throws Exception {
 			SSLContext sslContext = SSLContext.getInstance("TLS");
 			try (FileInputStream fis = new FileInputStream(Grasscutter.getConfig().getDispatchOptions().KeystorePath)) {
 				char[] keystorePassword = Grasscutter.getConfig().getDispatchOptions().KeystorePassword.toCharArray();
-
+				KeyManagerFactory _kmf;
 				try {
 					KeyStore ks = KeyStore.getInstance("PKCS12");
 					ks.load(fis, keystorePassword);
 					KeyManagerFactory kmf = KeyManagerFactory.getInstance("SunX509");
+					_kmf = kmf;
 					kmf.init(ks, keystorePassword);
 				} catch (Exception e) {
 					Grasscutter.getLogger().warn("[Dispatch] Unable to load keystore. Using default keystore password...");
 					KeyStore ks = KeyStore.getInstance("PKCS12");
 					ks.load(fis, "123456".toCharArray());
 					KeyManagerFactory kmf = KeyManagerFactory.getInstance("SunX509");
 					kmf.init(ks, "123456".toCharArray());
```

Comment:
Please consider adding another message notifying the user that the default password worked, and that they should probably change it in their config file.

Reply:
(GitHub didn't load the last commit while I was reviewing, so pretend this is on line 188)